packages/components/src/internal/components/editable/Controls.tsx

```diff
@@ -122,43 +119,26 @@ export class AddRowsControl extends React.Component<AddRowsControlProps, AddRows
         return count !== undefined && !this.isValid(count);
     }
 
-    onQuickAdd() {
-        const { onQuickAdd } = this.props;
-        if (onQuickAdd) {
-            onQuickAdd(this.state.count);
-        }
-    }
-
     renderButton() {
-        const { disable, quickAddText, onQuickAdd, addText, nounSingular, nounPlural, invalidCountMsg } = this.props;
+        const { disable, quickAddText, addText, nounSingular, nounPlural, invalidCountMsg } = this.props;
```

Comment:
Is `quickAddText` still used?  Did we never have these properties defined?

Reply:
We stopped using `quickAddText` so I removed it.